packages/graph-editor/src/editor/index.tsx

```diff
@@ -53,11 +53,6 @@ const snapGridCoords: SnapGrid = [16, 16];
 const defaultViewport = { x: 0, y: 0, zoom: 1.5 };
 const panOnDrag = [1, 2];
 
-const fullNodeTypes = {
```

Comment:
Initializing fullNodeTypes here causes custom node types to not be available to React Flow, so moving this inside the component code to be initialized as a ref later when the component is being mounted.